pkgs/development/libraries/libint/default.nix

```diff
@@ -52,23 +155,44 @@ let
 
     buildInputs = [ boost eigen ];
 
-    preConfigure = "./autogen.sh";
-
-    configureFlags = [
-      "--enable-eri=2"
-      "--enable-eri3=2"
-      "--enable-eri2=2"
-      "--with-eri-max-am=7,5,4"
-      "--with-eri-opt-am=3"
-      "--with-eri3-max-am=7"
-      "--with-eri2-max-am=7"
-      "--with-g12-max-am=5"
-      "--with-g12-opt-am=3"
-      "--with-g12dkh-max-am=5"
-      "--with-g12dkh-opt-am=3"
-      "--enable-contracted-ints"
-      "--enable-shared"
-    ] ++ lib.optional enableFMA "--enable-fma";
+    preConfigure = with lib.strings; ''
+      ./autogen.sh
```

Comment:
Isn't this automatically invoked in configurePhase?

Reply:
> Those are to many options and I am against merging this in the current state.
> 
> I would be fine with having an easier option to overwrite the configureFlags or a very small subset of this.

Certainly, and I would be a much in favour of handling less myself. Unfortunately libint is used in a large variety of variants and we are not the first bitten by it:

https://github.com/evaleev/libint/wiki
https://github.com/evaleev/libint/issues/190

Unfortunately again, it is a quite performance critical part of quantum chemistry.

While there are many options now I've went some way to ensure only reasonable combinations can be specified.

@markuskowa what is your opinion on this?